specification/compute/resource-manager/Microsoft.Compute/stable/2020-09-30/sharedGallery.json

```diff
@@ -0,0 +1,786 @@
+    },
+    "SharedGalleryImageVersionProperties": {
+      "properties": {
+        "publishedDate": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The published date of the gallery image version Definition. This property can be used for decommissioning purposes. This property is updatable."
+        },
+        "endOfLifeDate": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The end of life date of the gallery image version Definition. This property can be used for decommissioning purposes. This property is updatable."
+        }
+      },
+      "description": "Describes the properties of a gallery image version."
+    },
+    "CloudError": {
+      "x-ms-external": true,
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/ApiError"
+        }
+      },
+      "description": "An error response from the Compute service."
+    },
+    "ApiError": {
+      "properties": {
+        "details": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ApiErrorBase"
+          },
+          "description": "The Api error details"
+        },
+        "innererror": {
+          "$ref": "#/definitions/InnerError",
+          "description": "The Api inner error"
+        },
+        "code": {
+          "type": "string",
+          "description": "The error code."
+        },
+        "target": {
+          "type": "string",
+          "description": "The target of the particular error."
+        },
+        "message": {
+          "type": "string",
+          "description": "The error message."
+        }
+      },
+      "description": "Api error."
+    },
+    "ApiErrorBase": {
+      "properties": {
+        "code": {
+          "type": "string",
+          "description": "The error code."
+        },
+        "target": {
+          "type": "string",
+          "description": "The target of the particular error."
+        },
+        "message": {
+          "type": "string",
+          "description": "The error message."
+        }
+      },
+      "description": "Api error base."
+    },
+    "InnerError": {
+      "properties": {
+        "exceptiontype": {
+          "type": "string",
+          "description": "The exception type."
+        },
+        "errordetail": {
+          "type": "string",
+          "description": "The internal error message or exception dump."
+        }
+      },
+      "description": "Inner error details."
+    }
+  },
+  "parameters": {
+    "SubscriptionIdParameter": {
+      "name": "subscriptionId",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "Subscription credentials which uniquely identify Microsoft Azure subscription. The subscription ID forms part of the URI for every service call."
+    },
+    "LocationNameParameter": {
```

Comment:
Changing this name from 'LocationNameParamter' to 'LocationNameParameter' is a breaking change compared to the most recent preview version. 
We do not want breaking changes from the latest Preview version compared to the first GA version because it will cause pain to your customers. 

I have been told that no customers use this API by the Gallery team, so this is allowed for now.

Reply:
Are we comparing stable/2020-09-30 to stable/2019-12-01 or preview/2020-09-30? I am confused on this. Last time I got told that the comparison is between stable/2020-09-30 and stable/2019-12-01. I will submit the breaking change exception for review.